### ISSUE TITLE
Starting ElastiCube fails

### ISSUE THREAD

**Issue by kbbgl** [kbbgl/e2ewd#33]
```
13/06/2019 07:25:39.598 [main] INFO  tests.MainTest - Retrieving list of ElastiCubes...
13/06/2019 07:25:42.716 [main] INFO  tests.MainTest - No ElastiCubes in RUNNING mode.
13/06/2019 07:25:42.724 [main] INFO  cmd_ops.CmdOperations - Starting ElastiCube Sample ECommerce
13/06/2019 07:25:42.735 [main] ERROR cmd_ops.CmdOperations - 'C:\Program' is not recognized as an internal or external command,
13/06/2019 07:25:42.736 [main] ERROR cmd_ops.CmdOperations - operable program or batch file.
```

**Comment by kbbgl** [comment]
Issue caused by https://github.com/kbbgl/e2ewd/issues/31

**Comment by kbbgl** [comment]
Fixed in version 2.9.1
